content/400-pulse/200-getting-started.mdx

```diff
@@ -65,11 +66,11 @@ With Pulse enabled, proceed with these steps to integrate Pulse into your applic
 
 <Admonition>
 
-Pulse requires [Prisma Client](/orm/prisma-client) version `4.16.1` or higher and [`@prisma/extension-pulse`](https://www.npmjs.com/package/@prisma/extension-pulse) version `1.0.1` or higher
+Pulse requires [Prisma Client](/orm/prisma-client) version `4.16.1` or higher and [`@prisma/extension-pulse`](https://www.npmjs.com/package/@prisma/extension-pulse) version `1.0.1` or higher.
 
 </Admonition>
```

Comment:
Could we replace this with the new admonition syntax?

```
:::Tip

text here

:::

learn more here: https://docusaurus.io/docs/markdown-features/admonitions

Reply:
Somehow just seeing this comment now 🤔 suggestion: what about doing a separate PR where we do a full sweep and find/replace all `Admonition` components with the new syntax? It also feels like there are more of similar things that were introduced in Docusauros (e.g. frontmatter changes, line highlights in code blocks) that need to be documented in the style guide and also cleaned up throughout the docs. Let's do all of that in dedicated PR maybe?